### ISSUE TITLE
Login issue in Home Assistant v0.117.6

### ISSUE THREAD

**Issue by Vinz87** [bimmerconnected/bimmer_connected#259]
**Describe the issue**

Dec 09 00:51:21 raspberrypi hass[1592]: 2020-12-09 00:51:21 ERROR (SyncWorker_29) [bimmer_connected.account] Authentication failed. Maybe your password is invalid?
Dec 09 00:51:21 raspberrypi hass[1592]: 2020-12-09 00:51:21 ERROR (SyncWorker_29) [bimmer_connected.account] The BMW Connected Drive portal returned an error: UNKNOWN_ERROR (received status code 400 and expected 200).
Dec 09 00:51:21 raspberrypi hass[1592]: Traceback (most recent call last):
Dec 09 00:51:21 raspberrypi hass[1592]:   File "/srv/homeassistant/lib/python3.7/site-packages/bimmer_connected/account.py", line 99, in _get_oauth_token
Dec 09 00:51:21 raspberrypi hass[1592]:     expected_response=expected_response_code, post=True)
Dec 09 00:51:21 raspberrypi hass[1592]:   File "/srv/homeassistant/lib/python3.7/site-packages/bimmer_connected/account.py", line 152, in send_request
Dec 09 00:51:21 raspberrypi hass[1592]:     raise IOError(msg)
Dec 09 00:51:21 raspberrypi hass[1592]: OSError: The BMW Connected Drive portal returned an error: UNKNOWN_ERROR (received status code 400 and expected 200).

**Expected behavior**

**Home Assistant version (if used)**

v0.117.6
it was working on v0.116.4

**Screenshots**

**ConnectedDrive website**

If you see any issues with the BMW integration in Home Assistant (or when directly using the library), please make sure to first login to the BMW ConnectedDrive website of your country and check if you can successfully login and query the car status.
- [x] I can still successfully login to the BMW ConnectedDrive website and the car status is available there.
- [x] I have ConnectedDrive enabled for my vehicle.

**Your config.yaml**

```yaml
bmw_connected_drive:
  Vinz:
    username: USERNAME_BMW_CONNECTED_DRIVE
    password: PASSWORD_BMW_CONNECTED_DRIVE
    region: "rest_of_world"
````

**Number of cars**

If you have more than 1 car please mention that here.
- [ ] I have 2 or more cars linked to the ConnectedDrive account.
- [ ] I have a Mini vehicle linked to my account.

**Output of bimmer_connected fingerprint**

**Comment by Vinz87** [comment]
I don't know if this is solved in newer releases of this repository, not included in Home Assistant yet.
If so, how can I integrate this repository as a custom component in Home Assistant?

**Comment by gerard33** [comment]
This is fixed in HA 0.118. So you can update your HA to the latest version.

**Comment by Vinz87** [comment]
> This is fixed in HA 0.118. So you can update your HA to the latest version.

Thank you, I'll try. However, it is not mentioned in [v0.118 release notes](https://www.home-assistant.io/blog/2020/11/18/release-118/)

**Comment by j4ys0n** [comment]
Confirmed that it's fixed in 0.118.5
